src/config/routes.js

```diff
@@ -46,9 +46,11 @@ function redirectToHome(nextState, replace) {
     replace('/');
     return;
   }
-  client.isLoggedIn().then(() => {
-    replace('/');
-  });
+  client.isLoggedIn()
+    .then(() => {
+      replace('/');
+    })
+    .catch(() => {});
 }
```

Comment:
Hmm we'd get an `unhandled error` I guess both should redirect to '/' looking into it and then I'm going to merge